readme.md

```diff
@@ -29,6 +31,29 @@ Or you can run it from Maven directly using the Spring Boot Maven plugin. If you
 
 > NOTE: If you prefer to use Gradle, you can build the app using `./gradlew build` and look for the jar file in `build/libs`.
 
+
+## Terminology
+Spring-Boot: It is a web-based framework. It allows us to make business based apps efficiently. 
```

Comment:
The Spring Boot it's not just a web framework.

Reply:
https://en.wikipedia.org/wiki/Spring_Boot

Above you'll find the definition of spring boot :)